national-resources/estonia.md

```diff
@@ -0,0 +1,121 @@
+
+The Health Board provides further information regarding infectious diseases on its home page, such as [safety guidelines and recommendations](https://www.terviseamet.ee/et/nakkushaigused-menuu/tervishoiutootajale/nakkusohutuse-juhendid-ja-soovitused) or [interactive dashboard](https://www.terviseamet.ee/et/nakkus-haigused/tervishoiutootajale/nakkushaigustesse-haigestumine) for the infectious disease history in Estonia but the majority of the information is available only in Estonian.
+
+
+## Disease surveillance initiatives
+During the COVID-19 pandemic, the need for additional surveillance data became pressing. With scientists being able to ensure rapid development and implementation of new methodologies, the following initiatives were implemented in cooperation with the University of Tartu:
+
+* [__Wastewater surveillance__](https://ut.ee/en/node/113141) was set up in August 2020 [by the researchers of the University of Tartu](https://ut.ee/en/node/113141) (prof Tanel Tenson and Veljo Kisand) and taken over [by the Health Board](https://www.terviseamet.ee/et/reoveeseire-kaardirakendus) in 2022. Based on data from Estonia, it was observed that the viral load in wastewater increased before a rise in the number of infected people was observed. Furthermore, in certain places such as dormitories and prisons, the viral load increase was used to predict outbreaks. In conclusion, wastewater studies had an additional value when infection rates were on the rise or decline, but not during a plateau.
+* [__Prevalence of SARS-CoV-2 IgG antibodies.__](https://covid19dataportal.ee/en/data_types/serology/data/) The determination of antibodies allowed for a retrospective assessment of the outbreak's extent and is an important tool for assessing the extent of population immunity. In Estonia, two seroepidemiological studies were conducted in the spring and late summer of 2020 (Koro-Sero-EST 1 end 2). Moving forward, the determination of antibodies was incorporated into the prevalence study.
+* [__Prevalence study.__](https://ut.ee/en/content/study-prevalence-coronavirus-estonia) The primary objectives of active surveillance were to evaluate the widespread presence of SARS-CoV-2 and the prevalence of antibodies. The first stage of the surveillance study took place from April 22nd to 29th, 2020. Adults from all over Estonia were invited to participate in the study based on a random sampling. In total, 33 stages were carried out, each involving 2000 - 2500 participants. The percentage of infected individuals fluctuated from less than 1% during the first wave to over 10% during the fourth wave. Active surveillance provided the most objective overview of the extent of virus presence in Estonia. The first stages of the prevalence study have been published by [Uusküla et al 2022](https://doi.org/10.1016%2Fj.puhe.2022.02.004).
```

Comment:
"...place from April 22nd to 29th, 2020": change the date format to "...from 22-29 April 2020."